frontend/src/app/testQueue/TestCard/TestCard.tsx

```diff
@@ -39,25 +41,22 @@ export const TestCard = ({
   const navigate = useNavigate();
   const timer = useTestTimer(
     testOrder.internalId,
-    testOrder.deviceType.testLength
+    testOrder.deviceType.testLength,
+    Number(testOrder.timerStartedAt)
   );
   const organization = useSelector<RootState, Organization>(
     (state: any) => state.organization as Organization
   );
+  const [updateTestOrderTimerStartedAt] =
+    useUpdateTestOrderTimerStartedAtMutation();
 
   const [isCloseModalOpen, setIsCloseModalOpen] = useState(false);
 
   const [isExpanded, setIsExpanded] = useState(true);
 
   const testCardElement = useRef() as React.MutableRefObject<HTMLDivElement>;
 
-  useEffect(() => {
```

Comment:
Just moved to keep the timer code next to the timer useEffect and still have both useEffects next to each other for component structure